main.py

```diff
@@ -112,45 +125,60 @@ def run_diff(config, name, address, explorer_api_token, github_api_token):
     logger.report_table(report)
 
 
-def main():
-    logger.info("Welcome to Diffyscan!")
-    logger.divider()
-
-    logger.info("Loading API tokens...")
-    explorer_api_token = load_env("ETHERSCAN_TOKEN", masked=True)
-    github_api_token = load_env("GITHUB_API_TOKEN", masked=True)
-
-    contract_address = load_env("CONTRACT_ADDRESS", required=False)
-    contract_name = load_env("CONTRACT_NAME", required=False)
-
-    logger.divider()
-
-    logger.info("Loading config...")
-    config = load_config()
+def process_config(path: str):
+    logger.info(f"Loading config {path}...")
+    config = load_config(path)
 
-    if contract_address is not None:
-        if contract_name is None:
+    if CONTRACT_ADDRESS is not None:
```

Comment:
Let's get rid of this at all (CONTRACT_ADDRESS, CONTRACT_NAME from env)

Reply:
with pleasure